core-bundle/src/Resources/contao/library/Contao/Database.php

```diff
@@ -192,6 +192,18 @@ public function execute($strQuery)
 		return $this->prepare($strQuery)->execute();
 	}
 
+	/**
+	 * Execute a statement and return the number of affected rows
+	 *
+	 * @param string $strQuery The query string
+	 *
+	 * @return int The number of affected rows
+	 */
+	public function executeStatement(string $strQuery): int
```

Comment:
shouldn't we also allow for parameters and types like doctrine does?

Reply:
Don’t think so, this is merely a replacement for `->query()` that works with non-SELECT statements.
If you need parameters and types you should use the `database_connection` services instead anyways.